rotkehlchen/tests/utils/api.py

```diff
@@ -68,7 +68,7 @@ def assert_proper_response(
         response.headers["Content-Type"] == "application/json"
     )
     if status_code:
-        assert response.status_code == status_code
+        assert response.status_code == status_code, response.json()
```

Comment:
omission?

Reply:
Oufff, yeah, I use it for debugging, thks :)